From e8ab5856a116f6b7b9bd28781fcf2f685cc6645f Mon Sep 17 00:00:00 2001
From: Frediano Ziglio <fziglio@redhat.com>
Date: Fri, 25 May 2018 19:46:34 +0100
Subject: [PATCH 14/43] file_xfer: Remove too C syntax for C++

In C++ simply declaring the struct add the structure name to the global
namespace, no needs for additional typedef.

Signed-off-by: Frediano Ziglio <fziglio@redhat.com>
Acked-by: Christophe Fergeau <cfergeau@redhat.com>
---
 vdagent/file_xfer.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/vdagent/file_xfer.h b/vdagent/file_xfer.h
index 029d0e7..747c29c 100644
--- a/vdagent/file_xfer.h
+++ b/vdagent/file_xfer.h
@@ -21,7 +21,7 @@
 #include <map>
 #include "vdcommon.h"
 
-typedef struct FileXferTask {
+struct FileXferTask {
     FileXferTask(HANDLE _handle, uint64_t _size, const TCHAR* _name):
     handle(_handle), size(_size), pos(0) {
         // FIXME: should raise an error if name is too long..
@@ -36,7 +36,7 @@ typedef struct FileXferTask {
     TCHAR name[MAX_PATH];
 
     void cancel();
-} FileXferTask;
+};
 
 typedef std::map<uint32_t, FileXferTask*> FileXferTasks;
 
-- 
2.17.1

